### PR TITLE
libplctag: init at v2.3.5

### PR DESCRIPTION
**Pull request by storvik**
###### Motivation for this change

Add library libplctag. 

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/libraries/libplctag/default.nix [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+stdenv.mkDerivation rec {
+  pname = "libplctag";
+  version = "2.3.5";
+
+  nativeBuildInputs = [ cmake ];
+
+  src = fetchurl {
+    url = "https://github.com/libplctag/libplctag/archive/v${version}.tar.gz";
+    sha256 = "16azdxyl82js17xcav7v23zvi64w72181wylj489zim9x5h5sxsr";
+  };
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/libplctag/default.nix:16]:
```suggestion
  src = fetchurl {
    url = "https://github.com/libplctag/libplctag/archive/v${version}.tar.gz";
    sha256 = "16azdxyl82js17xcav7v23zvi64w72181wylj489zim9x5h5sxsr";
  };
  
  nativeBuildInputs = [ cmake ];
```

File: pkgs/development/libraries/libplctag/default.nix [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+  meta = with lib; {
+    homepage = "https://github.com/libplctag/libplctag";
+    description = "Library that uses EtherNet/IP or Modbus TCP to read and write tags in PLCs";
+    license = [ licenses.lgpl2Plus licenses.mpl20 ];
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/libplctag/default.nix:21]:
```suggestion
    license = with licenses; [ lgpl2Plus mpl20 ];
```

File: pkgs/development/libraries/libplctag/default.nix [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    maintainers = [ maintainers.petterstorvik ];
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/libplctag/default.nix:22]:
```suggestion
    maintainers = with maintainers; [ petterstorvik ];
```

File: pkgs/development/libraries/libplctag/default.nix [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libplctag";
+  version = "2.3.5";
+
+  nativeBuildInputs = [ cmake ];
+
+  src = fetchurl {
+    url = "https://github.com/libplctag/libplctag/archive/v${version}.tar.gz";
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/libplctag/default.nix:14]: Please use fetchFromGitHub.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by storvik** [comment]
Thanks for feedback @SuperSandro2000, changes have been committed.

File: pkgs/development/libraries/libplctag/default.nix [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+, fetchFromGitHub
+}:
+
+stdenv.mkDerivation rec {
+  pname = "libplctag";
+  version = "2.3.5";
+
+  src = fetchFromGitHub {
+    owner = "libplctag";
+    repo = "libpcltag";
```
Comment (SuperSandro2000) [inline pkgs/development/libraries/libplctag/default.nix:13]:
```suggestion
    repo = "libplctag";
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Pull request merged**